Lombiq.Tests.UI/Extensions/VisualVerificationUITestContextExtensions.cs

```diff
@@ -27,109 +29,241 @@ public static class VisualVerificationUITestContextExtensions
 to customize the name of the dump item.";
 
     /// <summary>
-    /// Compares the reference image and screenshot of the whole page. The mean error percentage should be less than or
-    /// equal to the given <paramref name="meanErrorPercentageThreshold"/>. The reference image is automatically
+    /// Compares the baseline image and screenshot of the whole page. The pixel error percentage should be less than or
+    /// equal to the given <paramref name="pixelErrorPercentageThreshold"/>. The baseline image is automatically
     /// loaded from assembly resource, if it doesn't exist exists then from the project path based on
-    /// <see cref="VisualVerificationMatchApprovedConfiguration"/> - it can be configured over <paramref name="configurator"/> -,
-    /// if the reference image doesn't exist, a new one will be created based on the element's screenshot, and an
-    /// <see cref="VisualVerificationReferenceImageNotFoundException"/> will be thrown. The reference image path is
-    /// generated from the method name - annotated with <see cref="VisualVerificationApprovedMethodAttribute"/> - and the source file
-    /// name and path, where the method is.
+    /// <see cref="VisualVerificationMatchApprovedConfiguration"/> - it can be configured over
+    /// <paramref name="configurator"/> -, if the baseline image doesn't exist, a new one will be created based on the
+    /// element's screenshot, and an <see cref="VisualVerificationBaselineImageNotFoundException"/> will be thrown. The
+    /// baseline image path is generated from the first method name - from the call stack which is not annotated with
+    /// <see cref="VisualVerificationApprovedMethodAttribute"/> - and the source file name and path, where the method is.
+    /// In case when you want visually validate elements that contain text on multiple platforms/browsers then keep in
+    /// mind that, the font rendering results different visuals. This means that you should use different baseline
+    /// images for each platform. You can generate baseline images for each platform with locally build and run tests
+    /// and follow the instructions in <see cref="VisualVerificationBaselineImageNotFoundException"/> or running on a CI
+    /// and using the image dumped on failure. Don't forget to configure the platforms with
+    /// <see cref="VisualVerificationMatchConfiguration&lt;TSelf&gt;.WithPlatforms"/> and suffixes as needed
+    /// <see cref="VisualVerificationMatchApprovedConfiguration.WithUsePlatformAsSuffix"/> and
+    /// <see cref="VisualVerificationMatchApprovedConfiguration.WithUseBrowserNameAsSuffix"/> using
+    /// <paramref name="configurator"/>.
```

Comment:
This is too long and complex for a single `summary`. Refactor it in the following way:

- The `summary` should be one-two sentences, explaining the point of the method.
- Add parameter-specific docs to the parameter documentation.
- If you feel that more general explanation is needed, at those as multiple `para`s in the `summary`.
- Use `<inheritdoc />` to DRY it with the other methods if possible.

Same for the other methods.

Reply:
I can't use `<inheritdoc/>` here, these methods are not overrides.

Reply:
Please try, I think it works in such cases too. If not, you can still use `see` like you did with the configuration variables and methods.